src/databrowser.js

```diff
@@ -138,11 +138,14 @@ define([
         }
         function handleDrop(node, info, target) {
             var widget = widgets.util.getWidget(target, vellum),
-                path = widget.mug.form.richText ? node.hashtag : node.xpath;
+                path = widget.mug.form.richText ? node.hashtag : node.xpath,
+                id;
             while (info) {
-                var id = widget.addInstanceRef({id: info.id, src: info.uri});
-                if (id !== info.id) {
-                    path = widget.mug.form.updateInstanceQuery(path, id, info.id);
+                if (info.id && info.uri) {
```

Comment:
It's hard to explain, but because of the `merge: true` flag, there is now an object in the `info._parent` hierarchy that does not have these attributes.

I had been trying to write a test to make this fail before adding `merge: true` to the test schemas, but was unable to. Once I added it, existing tests failed, and this fixes them.